### DISCUSSION TITLE
how to setup mpv+yt-dlp to play youtube 1080p with AV01 and 1440p with VP9?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14469] [Q&A] [answered]
Hi, I would like to play youtube videos that are max res 1080p with AV01 and videos that are above 1080p like 1440p with VP9.
How can I achieve this?

Thank you :)

**Comment by adisib** [comment, marked as answer]
Probably something like this (but I didn't test it):
`ytdl-format=bestvideo[height>=?1440][vcodec^=vp9]+bestaudio/bestvideo[height<=?1440][vcodec^=av0]+bestaudio/bestvideo+bestaudio/best`

**Reply by geextahslex** [reply]
This works, thank you.
My only question is how can I limit the max resolution to 1440p, because sometimes I don't want 4k

**Comment by Lavkush-Singh34** [comment]
> Hi, I would like to play youtube videos that are max res 1080p with AV01 and videos that are above 1080p like 1440p with VP9. How can I achieve this?
> 
> Thank you :)

```bash
mpv --ytdl-format="bv[height<=720]+ba" <yt_video_url>
```

```bash
mpv --ytdl-format="bv[height<=480]+ba" <yt_video_url>
```

You can change the resolution in place of 720 like 240,360,1080 even 1440.
